### ISSUE TITLE
Inquiry on Training Approach: Simultaneous vs. Separate Dataset Training for Your Model

### ISSUE THREAD

**Issue by liangyingshao** [GCYZSL/MoLA#23]
I have a question regarding the training approach of your model. Specifically, is your model trained simultaneously on multiple datasets (such as ScienceQA, CommonsenseQA, and OpenbookQA), or is it trained separately on each dataset? If it's the former, can MoE (Mixture of Experts) be considered as achieving an ensemble effect through the aggregation of results from different experts?
Thank you!

**Comment by GCYZSL** [comment]
Hi Liangying, for the first question, our model is trained separately. For the second question, from my perspective, you can definitely expect MoE to achieve an ensemble effect.

**Comment by liangyingshao** [comment]
Thank you for your response! Additionally,  I notice that in the training script you provided, the `val_set_size` is set to 1, which means the size of the validation set is 1. Did you also use this setting in your actual training? A validation set of this size seems unlikely to provide accurate evaluation results.

**Comment by GCYZSL** [comment]
Hi Liangying, thank you for your question. As mentioned in our paper, we did a grid search on the epochs with a val set if available. For the training code, we only saved the last checkpoint and did not use the validation set. Hence, the val_set_size is set to 1.

**Comment by liangyingshao** [comment]
Thanks for your responses.

**Comment by liangyingshao** [comment]
Thank you again for your work! I noticed that the paper states, "All experiments were conducted on the servers with eight A100-40G GPUs," but I didn't see any distributed code in the implementation. Could you clarify this?

**Comment by GCYZSL** [comment]
It is handled by Huggingface automatically.
